src/Language/PureScript/CodeGen/JS.hs

```diff
@@ -69,7 +69,7 @@ moduleToJs (Module coms mn imps exps foreigns decls) foreign_ =
     let standardExps = exps \\ foreignExps
     let exps' = JSObjectLiteral Nothing $ map (mkString . runIdent &&& JSVar Nothing . identToJs) standardExps
                                ++ map (mkString . runIdent &&& foreignIdent) foreignExps
-    return $ moduleBody ++ [JSAssignment Nothing (JSAccessor Nothing "exports" (JSVar Nothing "module")) exps']
+    return $ moduleBody ++ [JSAssignment Nothing (JSIndexer Nothing (JSStringLiteral Nothing "exports") (JSVar Nothing "module")) exps']
```

Comment:
Would it make sense to use `accessorString` here (and in the other places in this module) to make the code a bit more concise?